### PR TITLE
Handle status packets asynchronously

### PR DESCRIPTION
**Pull request by coelho**
Status packets are currently behind handled on the main thread. With this PR, they are moved off of the main thread and are handled by the NIO threads.

It is good to note however that ServerListPingEvent, although incorrectly documented, is an asynchronously fired event within currently available Bukkit implementations.

### CODE REVIEW

File: src/main/java/net/glowstone/net/message/status/StatusRequestMessage.java [hunk 1]
```diff
@@ -1,6 +1,12 @@
...
+import com.flowpowered.networking.AsyncableMessage;
 
-public final class StatusRequestMessage implements Message {
+public final class StatusRequestMessage implements AsyncableMessage {
+
+    @Override
+    public boolean isAsync() {
+        return true;
+    }
+
```
Comment (turt2live) [inline src/main/java/net/glowstone/net/message/status/StatusRequestMessage.java]: Whitespace

**Comment by turt2live** [comment]
Sorry - bot fail :(

**Comment by turt2live** [comment]
Looks good!

**Comment by SpaceManiac** [comment]
Merged in 0c6e09dd4a087b32f803e2ab14457a2a18daf82b.

**Pull request closed without merging**